intranet/apps/preferences/fields.py

```diff
@@ -25,7 +22,5 @@ def to_python(self, value):
         return value
 
     def widget_attrs(self, widget):
-        attrs = {}
-        if self.max_length is not None:
-            attrs.update({'maxlength': str(self.max_length)})
+        attrs = {"maxlength": "17"}
```

Comment:
I don't think so, I changed the mobile phone field to a phone number, verified it saved, and changed it back to blank, and after a refresh the field displays "None".